### ISSUE TITLE
Conflict with Linter

### ISSUE THREAD

**Issue by afrazkhan** [haad/language-Ansible#9]
This seems to conflict with [linter](https://atom.io/packages/linter) --> [linter-js-yaml](https://atom.io/packages/linter-js-yaml). When this is enabled, linting of YAML does not work.

**Comment by xenithorb** [comment]
I agree, this should be improved in such as way as to not clobber with all YAML files. 

Where this particularly annoys me right now is in having to write a lot of docker-compose.yml files.
